low-level/framework/sspl_ll_d

```diff
@@ -740,6 +738,9 @@ def refresh_iem_log_file(signal_number, frame):
     else:
         logger.info(f"IEMSensor is not in running, ignoring signal to refresh file")
 
+def generate_cov_report(signal_number, frame):
+    #DO NOT EDIT: Marker comment to dynamically add code to stop coverage, save and generate code coverage report
+    pass
 
```

Comment:
dont keep placeholder function too, just keep marker comments and add function code and signhandler registration code through separate start_code_coverage.sh script only

Reply:
addressed.